code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -1811,3 +1811,188 @@
 				continue
 			L.adjustSanityLoss(-heal_amount)
 			new /obj/effect/temp_visual/healing(get_turf(L))
+
+/obj/item/ego_weapon/blind_obsession//When I saw that Ishmael's version was an anchor I thought "hey would it be funny if it was a throwing weapon with aoe".
+	name = "blind obsession"
+	desc = "All hands, full speed toward where the lights flicker. The waves... will lay waste to everything in our way."
+	special = "This weapon requires two hands to use. \
+			Use in hand to unlock its full power for a short period of time at the cost of speed. \
+			Knocks certain enemies backwards when at full power. \
+			When at thrown at full power, this weapon damages everyone but yourself in an AOE. Be careful! \
+			This weapon deals 75% more damage on fully powered direct throws."
+	icon_state = "blind_obsession"
+	force = 80
+	attack_speed = 3
+	throwforce = 80//same as weapon
+	throw_speed = 5
```

Comment:
Change the throw speed to 0.5 to 1. 
Make it slow so people can dodge the gigantic nuke attack that one shots them